.gitignore

```diff
@@ -0,0 +1,5 @@
+*.DS_Store
+.DS_Store
```

Comment:
Line 2 will match `.DS_Store` in any directory; the other lines are unneeded.

Consider adding this to your global gitignore.